src/main/flight/pid.c

```diff
@@ -638,6 +638,21 @@ static void NOINLINE pidApplyFixedWingRateController(pidState_t *pidState, fligh
         pidState->errorGyroIf += rateError * pidState->kI * dT;
     }
 
+    // Calculate new D-term
```

Comment:
This is exactly the same as in MR controller. It would be better to extract Dterm handling to a separate static function and use in both FW and MR controller. 

If `dBoostFactor` will be set to 0 for FW controler, even the `applyDBoost` can stay unchanged

Reply:
Done, I left the dboost factor the same for FW as it is for MR for now. Needs to be tested.